src/test/synthetic_table_generator_test.cpp

```diff
@@ -24,12 +24,12 @@ TEST_F(SyntheticTableGeneratorTest, StringGeneration) {
 
 TEST_F(SyntheticTableGeneratorTest, ThrowOnParameterLengthMismatch) {
   auto table_generator = std::make_shared<SyntheticTableGenerator>();
-  const auto uniform_distribution = ColumnDataDistribution::make_uniform_config(0.0, 1.0);
+  auto uniform_distribution_0_1 = ColumnDataDistribution::make_uniform_config(0.0, 1.0);
+  std::vector<ColumnSpecification> column_specs = {size_t{5}, {uniform_distribution_0_1, DataType::Double}};
```

Comment:
I think you can use `column_specs{5, {uniform_distribution_0_1, DataType::Double}}` ... 
Besides, I am not really a fan of using curly brackets for constructor calls. This post summarizes my personal opinion quite well: https://stackoverflow.com/a/37228443/1147726. Not sure if there is a common accepted pattern when to use what.

Reply:
@mrks : we don't have a guideline for Hyrise here, right?

Reply:
I was a fan of curly braces until I spent half an hour debugging a case where I used `vector{a, b}` and ended up with a vector of size 2. I still used curly braces (and [auto-to-stick](https://www.fluentcpp.com/2018/09/28/auto-stick-changing-style/), but avoid it for vector initializations). No hard rules here.

Reply:
So, what's the agreement here? This is the last open discussion I think...

Reply:
As the post https://stackoverflow.com/a/37228443/1147726 recommends, I'd use `()` for constructor calls and `{}` for passing a data list. So I'd vote for `()` here, since it's calling a "complex" constructor and does not pass a data item of 5 here.

Reply:
I'd do `auto column_specs = std::vector<ColumnSpecification>(...);`, but I don't think we should be blocked by this. @Bouncner, ready to go?

Reply:
Yep, ready to go with that change.

Reply:
Mmh, I was just about to change this myself so that we can merge this, but I don't have any rights on the PR. :/

Reply:
@mrks you have been granted the rights to our fork but you probably need to accept it here https://github.com/bakoe/hyrise/invitations

Reply:
RE this change: I just realized that including the encoding spec in our column specification makes this test case unnecessary so it's actually removed in https://github.com/hyrise/hyrise/pull/1987/commits/710bf3f25a8a0baa40682daa5d7426cb1de64b6e

Reply:
Haha, ok